R/descriptives.R

```diff
@@ -380,7 +377,23 @@ DescriptivesInternal <- function(jaspResults, dataset, options) {
   return()
 }
 
-.descriptivesDescriptivesTable <- function(dataset, options, jaspResults, numberMissingSplitBy = 0) {
+.descriptivesReadData <- function(options, variables, splitName) {
+
+  makeSplit <- splitName != ""
+  dataset <- .readDataSetToEnd(columns.as.numeric = variables, columns.as.factor = if (makeSplit) splitName)
+  dataset.factors <- jaspBase::readDataSetByVariableTypes(options, "variables")
+  if (makeSplit)
+    dataset.factors[[splitName]] <- dataset[[splitName]]
+
+  allMissing <- \(x) all(is.na(x))
```

Comment:
very fancy!

Reply:
Die is er al een tijdje ;), zie https://developer.r-project.org/blosxom.cgi/R-4-1-branch/NEWS/2021/05/16#n2021-05-17. De korte versie vind ik wat fijner voor (l)apply enzo.